lib/CMakeLists.txt

```
@@ -4,7 +4,10 @@
 
 # Library: concurrentqueue {{{
 
-list(APPEND dirs ${CMAKE_CURRENT_LIST_DIR}/concurrentqueue/include)
+add_library(concurrentqueue INTERFACE)
+target_include_directories(concurrentqueue INTERFACE
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/concurrentqueue/include>)
```

Also here, two spaces

```suggestion
  $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/concurrentqueue/include>)
```